composeml/label_maker.py

```diff
@@ -77,11 +77,14 @@ def search(self, df, minimum_data, num_examples_per_instance, gap, verbose=True,
         assert_valid_offset(gap)
         name = self.labeling_function.__name__
 
-        def df_to_labels(df):
+        def df_to_labels(df, progress_bar=None):
+            labels = pd.Series(name=name)
             df = df.loc[df.index.notnull()]
             df.sort_index(inplace=True)
 
-            labels = pd.Series(name=name)
+            if progress_bar is not None:
+                not_done = progress_bar.n < progress_bar.total
+                progress_bar.update(n=int(not_done))
```

Comment:
why does this get called if the progress bar is already completed?

Reply:
should the update method get called at the end of this function onces it's actually completed?

also, we can update more fine grained after every iteration of `for example in range(num_examples_per_instance):`

basically `progress_bar.update(1/(ngroups * num_examples_per_instance))`

Reply:
That's correct. I fixed that. Also, change so that the progress bar for each example. Defined the total as `ngroups * num_examples_per_instance`.